src/skadi/sample.cc

```diff
@@ -35,10 +34,43 @@ constexpr int16_t NO_DATA_HIGH = 16384;
 constexpr int16_t NO_DATA_LOW = -16384;
 constexpr size_t TILE_COUNT = 180 * 360;
 constexpr int8_t UNPACKED_TILES_COUNT = 50;
+const constexpr char* kDataPathPattern = "{DataPath}";
 
-// macro is faster than inline function for this..
+// macro is faster than inline function for this...
 #define out_of_range(v) v > NO_DATA_HIGH || v < NO_DATA_LOW
 
+template <typename T>
+void parallel_call(const std::function<double(const T&)>& func,
```

Comment:
nit: the name is too generic

Reply:
well... it was done intentionally. As this function is just a utility function i tried to keep it as general as possible. i thought it might be good idea to move it to a separate unit and so to make it available to other library modules and reused if needed (probably not with this pr). what do you think?

Reply:
i can rename of course if you think it is better

Reply:
removed parallel_call